pmm-app/src/pmm-qan/panel/QueryAnalytics.tsx

```diff
@@ -97,15 +109,20 @@ const QueryAnalyticsPanel: FC<QueryAnalyticsPanelProps> = ({ grafanaTheme }) =>
           <div className={styles.splitterWrapper}>
             {/* @ts-ignore */}
             <SplitPane
+              key={querySelected ? 'split' : 'full'}
```

Comment:
Do we need to change the key if `querySelected` is changed? Since it re-renders the whole component, do the new values for pane1Styles/pane2Style not do enough?